lib/persisted-model.js

```diff
@@ -1100,6 +1100,10 @@ module.exports = function(registry) {
       since = { source: since, target: since };
     }
 
+    if (options && typeof options === 'function') {
```

Comment:
When `options` are not set, then `typeof options` is `undefined`.I am proposing to rework this into:

```js
if (typeof options === 'function') {
  options = {};
}
```